public/app/features/explore/Explore.tsx

```diff
@@ -302,6 +311,27 @@ export class Explore extends React.PureComponent<Props, ExploreState> {
     );
   }
 
+  isNodeGraphEnabled(datasource: DataSourceApi): boolean {
```

Comment:
Do we actually need all this? It feels like simple `datasource.nodeGraph?.enabled? === undefined || datasource.nodeGraph?.enabled? === true` would do the same thing wouldn't it?

Reply:
Or maybe it would make sense to use something like @ifrost introduced here https://github.com/grafana/grafana/pull/39536? We could still keep the interface private/alpha but would then make it easier to see if data source implements the capability and then just call some simple `isNodeGraphEnabled` method or something like that.